_overviews/scala3-book/control-structures.md

```diff
@@ -28,30 +28,76 @@ These are all demonstrated in the following sections.
 
 A one-line Scala `if` statement looks like this:
 
+{% tabs control-structures-1 class=tabs-scala-version %}
+{% tab 'Scala 2' for=control-structures-1 %}
+```scala
+if (x == 1) println(x)
+```
+{% endtab %}
+{% tab 'Scala 3' for=control-structures-1 %}
 ```scala
 if x == 1 then println(x)
 ```
+{% endtab %}
+{% endtabs %}
 
 When you need to run multiple lines of code after an `if` equality comparison, use this syntax:
 
+{% tabs control-structures-2 class=tabs-scala-version %}
+{% tab 'Scala 2' for=control-structures-2 %}
+```scala
+if (x == 1) {
+  println("x is 1, as you can see:")
+  println(x)
+}
+```
+{% endtab %}
+{% tab 'Scala 3' for=control-structures-2 %}
 ```scala
 if x == 1 then
   println("x is 1, as you can see:")
   println(x)
 ```
+{% endtab %}
+{% endtabs %}
 
 The `if`/`else` syntax looks like this:
 
+{% tabs control-structures-3 class=tabs-scala-version %}
+{% tab 'Scala 2' for=control-structures-3 %}
+```scala
+if (x == 1) {
+  println("x is 1, as you can see:")
+  println(x)
+} else
+  println("x was not 1")
```

Comment:
It might be a matter of taste, but I usually also add the braces to the `else` branch even if it’s a one-liner, if there are braces to the “then” branch.